src/components/label/Label.tsx

```diff
@@ -32,7 +34,16 @@ export interface LabelProps {
     tooltip?: React.ReactNode;
 }
 
-const Label = ({ text, tooltip, infoTooltip, infoIconProps, showOptionalText, hideLabel, children }: LabelProps) => {
+const Label = ({
+    text,
+    tooltip,
+    infoTooltip,
+    infoIconProps,
+    showOptionalText,
+    hideLabel,
+    children,
+    labelElProps = {},
```

Comment:
nit: should be okay to omit the default value `{}` since the prop type in `StandardLabel` and `LabelPrimitive` is optional and the spread operator in `{...labelElProps}` will handle `undefined`